README.md

```diff
@@ -17,17 +21,14 @@ The following dependencies must be available:
 
 ## 1. User & IAM
 
-* The user account should have access to the GCP project for perform operation.
-* Service account or user credentials with the following privileged roles must be used to provision the resources of this module:
+* The principal executing the TF should have following permissions
 
    * Service Account Admin
    * IAM Workload Identity Pool Admin
    * Project IAM Admin
 
 ## 2. Install terraform
 
-This module is meant for use with Terraform version = "~> 3.61.0".
```

Comment:
If we are removing then better to mention like below because people may have old version of TF which can cause problem .
## 2. Install terraform ( preferred latest )

Reply:
I would assume "latest" is obvious unless we don't work with some old versions.
Also, the original test was wrongly referring to the Google's TG module version as TF's version.
This caused some confusion with customers who tried it.
They were looking for 3.61.* for TF which is not going to be found.